src/plugins/es_ui_shared/public/index.ts

```diff
@@ -18,3 +18,12 @@
  */
 
 export * from './components/json_editor';
+
+export {
```

Comment:
pinging @cjcenizal & @alisonelizabeth on these since I saw your names in the git logs